docs/deploying-cf-aws.MD

```diff
@@ -0,0 +1,374 @@
+The following command will configure the environment where the bosh director and the jumpbox VMs are instantiated and configured with the `bbl` command:
+
+```bash
+bbl plan \
+    --aws-access-key-id $access_key_id \
+    --aws-secret-access-key $access_key_secret \
+    --aws-region $region \
+    --iaas aws \
+    --lb-type cf \
+    --lb-cert cert.pem \
+    --lb-key key.pem \
+    --lb-domain $SYSTEM_DOMAIN \
+    --name $ENV_NAME
+```
+
+The following command will instruct `bbl` to execute the configured enviroment defined in the previous command. It is possible to combine both commands so that the `plan` command is no longer necessary and just running `bbl up ...` with the same parameters in the `plan` command is enough.
+
+```bash
+bbl up \
+    --aws-access-key-id $access_key_id \
+    --aws-secret-access-key $access_key_secret \
+    --aws-region $region \
+    --iaas aws \
+    -n
+```
+
+Note that this command takes about 30 minutes to complete. Once completed, run the following command to ensure that the bosh director has been successfully deployed:
+
+```bash
+eval "$(bbl print-env)"; bosh log-in
+```
+
+You should see something like this:
+
+```bash
+$> eval "$(bbl print-env)"; bosh log-in
+Successfully authenticated with UAA
+
+Succeeded
+```
+
+## Deploy Cloud Foundry in AWS
+We're half way through, at this point there should be 2 EC2 instances in AWS that are the jumpbox and the bosh director. The `vars/` subdirectory contains the terraform and other sensitive files to connect to the bosh director as well as to authenticate through the jumpbox.
+
+### Clone the cf-deployment repository:
+Before installing Cloud Foundry, you will need to clone the cf-deployment repository, which contains the configuration to setup Cloud Foundry in a development environment:
+
+```
+git clone https://github.com/cloudfoundry/cf-deployment.git
+```
+
+### Validate OS Stemcell based on cf-deployment
+
+The OS stemcell defined in bbl might not match the one in the cf-deployment.yaml. During the bosh director deployment, bbl will upload its preconfigured version, but it might not match the one defined in the `cf-deployment/cf-deployment.yml` file. Check the output from the `bbl up` command and locate the line where it mentions which stemcell version is uploading:
+
+```bash
+...
+Finished installing CPI (00:03:30)
+
+Uploading stemcell 'bosh-aws-xen-hvm-ubuntu-jammy-go_agent/1.822'... Finished (00:04:23)
+
+Started deploying
+...
+```
+
+Compare it with the one in the `cf-deployment/cf-deployment.yml` file:
+
+```
+tail -n4 cf-deployment/cf-deployment.yml
+```
+
+Example:
+```
+$> tail -n4 cf-deployment/cf-deployment.yml
+stemcells:
+- alias: default
+  os: ubuntu-jammy
+  version: "1.824"
+```
+
+If they don't match, upload the stemcell specified in the cf-deployment.yml. The following command is also retrieveable from the bosh.io website https://bosh.io/stemcells/bosh-aws-xen-hvm-ubuntu-jammy-go_agent )
```

Comment:
nit: maybe `[bosh.io website](https://bosh.io/stemcells/bosh-aws-xen-hvm-ubuntu-jammy-go_agent)` to ease reading

Reply:
Done